package.json

```diff
@@ -45,6 +45,9 @@
     "react": "*",
     "react-test-renderer": "*"
   },
+  "files": [
```

Comment:
The "files" property is very dangerous; if you want to exclude things, use .npmignore.

That said, src and tests should absolutely be included - `npm explore foo && npm install && npm test` should always work - so there's not likely to be much worth ignoring.